client/js/verification-portal/form.js

```diff
@@ -229,7 +261,7 @@ class VerificationPortalForm extends Component {
 					</div>
 					<div className={classes.parent}>
 						<Field
-							label="Services Verified"
+							label="Verified"
```

Comment:
We can remove this label. The "Verified" label is only listed once, as a heading at the top, on the new designs.